modules/tinymce/src/themes/silver/main/ts/ui/dialog/Tree.ts

```diff
@@ -10,10 +10,41 @@ import * as Icons from '../icons/Icons';
 type TreeSpec = Omit<Dialog.Tree, 'type'>;
 type OnLeafAction = (id: string) => void;
 
+interface RenderItemProps {
+  backstage: UiFactoryBackstage;
+}
+
+interface RenderLeafLabelProps extends RenderItemProps {
+  leaf: Dialog.Leaf;
+  visible: boolean;
+  treeId: string;
+  onLeafAction: OnLeafAction;
+}
+
+interface RenderDirectoryProps extends RenderItemProps {
+  directory: Dialog.Directory;
+  labelTabstopping: boolean;
+  treeId: string;
+  onLeafAction: OnLeafAction;
+}
+
+interface RenderDirectoryLabelProps extends RenderItemProps {
+  directory: Dialog.Directory;
+  visible: boolean;
+  noChildren: boolean;
+}
+
+interface RenderDirectoryChildrenProps extends RenderItemProps {
+  children: Dialog.TreeItem[];
+  visible: boolean;
+  treeId: string;
+  onLeafAction: OnLeafAction;
+}
+
 const renderLabel = (text: string ): SimpleSpec => ({
   dom: {
     tag: 'span',
-    classes: [ `tox-tree__label` ],
+    classes: [ 'tox-tree__label' ],
```

Comment:
Should the classes used in this file for the tree component get put in a single object, say `TreeClasses` and keep calling this object instead of using strings everywhere ?